content/sensu-go/5.16/guides/monitor-sensu.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "Monitor Sensu with Sensu"
+linkTitle: "Monitor Sensu with Sensu"
+description: "Make sure your Sensu components are properly monitored. This guide describes best practices and strategies for monitoring Sensu."
+weight: 230
+version: "5.16"
+product: "Sensu Go"
+platformContent: False
+menu: 
+  sensu-go-5.16:
+    parent: guides
+---
+
+
+- [Monitor your Sensu backend instances](#monitor-your-sensu-backend-instances)
+- [Monitor your Sensu agent instances](#monitor-your-sensu-agent-instances)
+- [Monitor your Sensu API instances](#monitor-your-sensu-api-instances)
+- [Monitor your Sensu dashboard instances](#monitor-your-sensu-dashboard-instances)
+
+This guide describes best practices and strategies for monitoring Sensu with Sensu. 
+It explains how to ensure your Sensu components are properly monitored, including your Sensu backend, agent, API, and dashboard instances.
+
+To completely monitor a Sensu stack (Sensu backend, agent, API, and dashboard), you will need to have at least one other independent Sensu instance.
+A single Sensu stack cannot monitor itself completely because if some components are down, Sensu cannot create events.
+
+This guide requires Sensu plugins.
+For more information about using Sensu plugins, see [Install plugins with assets][16].
+
+_**NOTE**: This guide assumes you are not using Sensu clustering. The strategies in this guide are still useful for monitoring cluster members, but you should follow [Run a Sensu cluster][17] to effectively monitor clustered instances._
+
+## Monitor your Sensu backend instances
+
+The host running the `sensu-backend` should be monitored in two ways:
+
+* Locally by a `sensu-agent` process for operating system checks/metrics and Sensu services that are not part of the Sensu event system (like the dashboard)
+* Remotely from an entirely independent Sensu stack to monitor Sensu components that must be running for Sensu to create events. 
+
+### Monitor the Sensu backend locally
+
+Monitor the host that the `sensu-backend` process runs on like any other node in your infrastructure.
+This includes checks and metrics for [CPU][1], [memory][2], [disk][3], and [networking][4].
+Find more plugins at [Bonsai, the Sensu asset index][5].
+
+### Monitor the Sensu backend remotely
+
+Monitor the `sensu-backend` server process from an independent Sensu stack.
+To do this, call Sensu's [health API endpoint][6] and use the [check-http plugin][7] with the following check definition:
+
+{{< highlight json >}}
+{
+  "type": "CheckConfig",
+  "api_version": "core/v2",
+  "metadata": {
+    "namespace": "default",
+    "name": "check_sensu_server_port"
+  },
+  "spec": {
+    "command": "check-http.rb -h remote-api-hostname -P 4567 -p /health?consumers=1 --response-code 204",
+    "subscriptions": [
+      "monitor_remote_sensu_api"
+    ],
+    "interval": 60,
+    "publish": true
+  }
+}
+{{< /highlight >}}
+
+## Monitor your Sensu agent instances
+
+**CONTENT NEEDED**
```

Comment:
* service checks
* port checks (if agent api is enabled)

Reply:
Outside of the API port, you can't really monitor the agent as if the agent is down, as a service check for it won't run. The keepalive is a built in monitor for the service being up and functional, at least partially (as network issues can cause a keepalive to go non-ok but the agent is fine).

Reply:
I added this content.